regulations/static/regulations/css/less/base-variables.less

```diff
@@ -78,7 +78,7 @@ variables.less contains all theme variable / variable overrides
  ----------------------
 */
 
-@body_font: Georgia, serif;
+@body_font: "Merriweather", Georgia, serif;
```

Comment:
Not for now, but we probably want to replace this variable altogether and just use the mixins.